src/auth/jwt.refreshtoken.strategy.ts

```diff
@@ -0,0 +1,34 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, ExtractJwt } from 'passport-jwt';
+import { Injectable, UnauthorizedException, Body } from '@nestjs/common';
+import { UsersService } from '../users/users.service';
+import { jwtConstants } from './constants';
+ 
+@Injectable()
+export class JwtRefreshTokenStrategy extends PassportStrategy(Strategy,"jwt-refreshtoken") {
+  constructor(private userService:UsersService) {
+    super({
+      jwtFromRequest: ExtractJwt.fromBodyField('access_token'),
```

Comment:
similar to jwt, this should be sent via auth headers :      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),